src/components/organisms/Carousel.vue

```diff
@@ -1,60 +1,58 @@
 <template>
 	<div class="carousel">
-		<CarouselElement :imageContainer="imageContainer" :key="imageContainer.id" v-for="imageContainer in imageContainers"/>
-		<CarouselNavigation v-on:traverse="switchImage" :arrows="arrowContainer"/>
+		<CarouselElement :imageContainer="imageContainer" :key="index"  v-for="(imageContainer, index) in carousel.imageContainers"/>
```

Comment:
what is carousel element? is it image, is it buttons, is it pagination or is just some tehnical component?

Reply:
It contains image and text